c/doc/devbox_setup.md

```diff
@@ -108,12 +109,46 @@ This script uses **cmake** to make a folder called "cmake" in your home director
 
  > Note: you will not be able to run the samples until you configure them with a valid IoT Hub device connection string. For more information, see [Run sample on Linux](run_sample_on_desktop_linux.md).
 
+ <a name="windowsce"/>
+ ## Set up a Windows Embedded Compact 2013 development environment
+
+ - Install [Visual Studio 2013][visual-studio]. You can use the free Community Edition if you meet the licensing requirements.
+ Be sure to include Visual C++ and NuGet Package Manager.
+ - Install [Application Builder for Windows Embedded Compact 2013][application-builder]
+ - Install [Toradex Windows Embedded Compact 2013 SDK][toradex-CE8-sdk].
```

Comment:
You need to have a Windows Embedded Compact 2013 SDK installed. Current version builds using Toradex one, I can add instruction to adapt the build files to a different one, but you still need to install at least one SDK to be able to build the code.